### ISSUE TITLE
Direct Reaction Product Prediction

### ISSUE THREAD

**Issue by TeresaZhou0127** [MolecularAI/Chemformer#41]
Hi Chemformer team,

I am using your Chemformer model for predicting the products of some reactions. However, I am a bit confused that if it's proper for me to use the same model for direct reaction prediction and retrosynthesis prediction. I didn't find any params that I can set for distinguish between the two downstream tasks. Can you please help me confirm what is the correct way for me to get the predicted reaction product with Chemformer? 

Thank you so much for your help!

Best

**Comment by EBjerrum** [comment]
There used to be different pretrained weights for the different finetuned tasks.

**Comment by TeresaZhou0127** [comment]
Thank you so much for your reply! Can you please give me some hints about where to find these weights and how to use them for different downstream task? Since I am fine-tuning chemformer for direct reaction prediction, but I found the predicted products kind of a mixture of both direct and retro-synthesis. Thank you again for your help!

**Comment by anniewesterlund** [comment]
Hi! The link to the weights trained in the first Chemformer paper is found in the first paragraph in the README file: "The public models and datasets available [here](https://az.box.com/s/7eci3nd9vy0xplqniitpk02rbg9q2zcq).". The weights are under `models/fine-tuned`: `uspto_50` is the the model trained for retrosynthesis, while `uspto_sep` and `uspto_mixed` are for forward prediction (predicting products given reactants). See the paper for more info on the difference between `uspto_sep`and `uspto_mixed`.

**Comment by EBjerrum** [comment]
Yes, the pretrained models are exactly the starting point for further finetuning. Pretraining was done with Smiles2smiles tasks with masking and smiles data augmentation, not reaction data. You can train from scratch on reaction data, if that's what you want, but the benefits of using the pretrained model as a starting point is quite large in terms of training speed and accuracy in top-1 prediction.

**Comment by anniewesterlund** [comment]
Closing due to inactivity
